resource-managers/kubernetes/integration-tests/README.md

```diff
@@ -13,7 +13,7 @@ directory:
 
     ./dev/dev-run-integration-tests.sh
 
-To run tests with Java 11 instead of Java 8, use `--java-image-tag` to specify the base image.
+To run tests with Java 21 instead of Java 17, use `--java-image-tag` to specify the base image.
```

Comment:
Thank you. However, new sentence is also wrong because we are running with Java 21 already. K8s-base Docker image is Java 21.

https://github.com/apache/spark/blob/733be49a8078b70d44b26c392a15218c73f290e7/resource-managers/kubernetes/docker/src/main/dockerfiles/spark/Dockerfile#L17

Reply:
Since we are using Java 21 already, I have updated the documentation which points how to use a different Java version instead of Java 21.